doc/nrf/ug_thread_tools.rst

```diff
@@ -191,7 +191,7 @@ Running OTBR using Docker
 
 For development purposes, you can run the OpenThread Border Router on any Linux-based system using a Docker container that already has the Border Router installed.
 This solution can be used when you are only interested in direct communication between your Border Router and the Thread network.
-For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as the :ref:`Python Controller for Matter <ug_matter_configuring>`) and an application running on a Thread node.
+For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as the :ref:`CHIP Tool Matter controller <ug_matter_configuring>`) and an application running on a Thread node.
```

Comment:
I think we use a bunch of different names to name the same thing. How about just: "CHIP Tool for Linux or macOS"?